.travis.yml

```diff
@@ -30,3 +30,17 @@ before_script:
   - zsh --version
 script:
   - scripts/tests.sh
+jobs:
+  include:
+    - stage: deploy gitbook
+      if: branch = master AND type = push
+      script:
+        - npm install --only=dev --ignore-scripts
+        - npm run docs:build
+      deploy:
+       provider: pages
+       skip-cleanup: true
+       github-token: $GITHUB_TOKEN
```

Comment:
@salmanulfarzy have you generated `GITHUB_TOKEN`?

Reply:
`TOKEN` is configured on travis settings and I've been using similar setup on my fork.

Reply:
I've included my token with `public_repo` access as `GITHUB_TOKEN` to travis environment variable section.